save-backend/build.gradle.kts

```diff
@@ -22,6 +23,9 @@ dependencies {
     implementation(project(":save-common"))
     implementation("org.liquibase:liquibase-core:${Versions.liquibase}")
     implementation("org.hibernate:hibernate-core:${Versions.hibernate}")
+    implementation("org.springframework.boot:spring-boot-starter-data-jpa:${Versions.springBoot}")
+    implementation("com.h2database:h2:${Versions.h2}")
```

Comment:
Shouldn't it be `testImplementation`?

Reply:
H2 console configuration is using it, so I guess it can't be.

Reply:
Ok. In the future we should move it to test sources, but until we properly support MySQL in main sources let's leave it as is.